comfy_cli/command/build_push.py

```
@@ -0,0 +1,304 @@
+        node["localDigest"] = digest
+        node["localSizeBytes"] = size
+        if not _has_text(node, "blobId"):
+            name = str(node.get("name") or f"node-{index}")
+            uploads.append(
+                PushUpload("node_zip", "customNodes", index, f"{name}.zip", digest, size, package.archive_path)
+            )
+
+    return PushPreparation(prepared, tuple(uploads), tuple(stale_source_uris), tuple(skipped_symlinks))
+
+
+def pending_uploads(preparation: PushPreparation) -> tuple[PushUpload, ...]:
```

Pylint (4.0.7)

[convention] Missing function or method docstring

(C0116)